api/resources/translations/messages-es.properties

```diff
@@ -559,36 +568,36 @@ contact.type.person.new = Nueva persona
 contact.type.person.plural = Personas
 contact.type.place.edit = Editar lugar
 contact.type.place.new = Nuevo lugar
-contact.type.wrong =
+contact.type.wrong = Tipo incorrecto, el contacto no es una persona.
 contact.updated = Contacto actualizado
 contacts.imported = Los contactos se han importado correctamente
-contacts.results.sort =
-contacts.results.sort.alpha = 
-contacts.results.sort.date.last.visited =
-contacts.results.sort.title = 
-contacts.visits.count =
-contacts.visits.visits = 
-database_closed.description = Un error inesperado ha ocurrido y la aplicación debe ser recargada.
+contacts.results.sort = Ordenar resultados
+contacts.results.sort.alpha = Alfabéticamente
+contacts.results.sort.date.last.visited = Por fecha de la última visita
+contacts.results.sort.title = Ordenar esta lista
+contacts.visits.count = {{count}}
+contacts.visits.visits = {VISITS, plural, una{visit} otra{visits}}
+database_closed.description = Se produjo un error inesperado y se debe recargar la aplicación.
 database_closed.title = Error inesperado
-date.from = De
-date.incorrect.advice = Corrija la fecha y hora de su dispositivo.
-date.incorrect.confirm = Aceptar
-date.incorrect.description = La fecha y la hora parecen estar configuradas incorrectamente en este dispositivo. Tener la fecha o la hora incorrecta significa que sus datos estarán dañados y puede significar que se pierda los objetivos.
+date.from = Desde
+date.incorrect.advice = Corrija la configuración de fecha y hora de su dispositivo.
+date.incorrect.confirm = Ok
+date.incorrect.description = La fecha/hora parece estar configurada incorrectamente en este dispositivo. Tener una fecha u hora incorrecta significa que sus datos estarán corruptos y puede significar que no alcance los objetivos.
 date.incorrect.expected = Fecha esperada
 date.incorrect.reported = Fecha reportada
 date.incorrect.title = Configuración de fecha incorrecta
-date.time.title =
-date.to = A
-days = Dias
+date.time.title = Fecha y hora
+date.to = Hasta
+date_filter.error.from_date = La fecha "Desde" no puede ser posterior a la fecha "Hasta".
+date_filter.error.to_date = La fecha "Hasta" no puede ser anterior a la fecha "Desde".
+days = dias
 daysoverdue = Días desde la visita del paciente
-ddoc.version = DDoc Versión
+ddoc.version = Revisión del webapp en el servidor
```

Comment:
Why `Revisión` and not `Versión`?

Reply:
The English file says; `Server webapp revision` 
I think revision is different than version. But I'm not 100% sure.

Reply:
I believe it's best to keep the original word `revision` to match the English file.